lms/djangoapps/bulk_email/tests/test_api.py

```diff
@@ -6,15 +6,23 @@
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory
 
+from common.djangoapps.course_modes.models import CourseMode
 from common.djangoapps.student.tests.factories import InstructorFactory
-from lms.djangoapps.bulk_email.api import create_course_email, get_course_email
+from lms.djangoapps.bulk_email.api import (
+    create_course_email,
+    determine_targets_for_course_email,
+    get_course_email,
+    update_course_email
+)
 from lms.djangoapps.bulk_email.data import BulkEmailTargetChoices
+from lms.djangoapps.bulk_email.models import CourseEmail
+from openedx.core.djangoapps.course_groups.models import CourseUserGroup
 from openedx.core.lib.html_to_text import html_to_text
 
 LOG_PATH = "lms.djangoapps.bulk_email.api"
 
 
-class CreateCourseEmailTests(ModuleStoreTestCase):
+class BulkEmailApiTests(ModuleStoreTestCase):
     """
     Tests for the `create_course_email` function of the bulk email app's public Python API.
```

Comment:
[nit] Can this dosctring be generalized like the test class title?

Reply:
Thanks for catching all the stale documentation pieces. I'll update this too.